khal/cli.py

```diff
@@ -93,6 +93,12 @@ def multi_calendar_option(f):
     return d(a(f))
 
 
+def no_mouse_option(f):
+    o = click.option('--no-mouse', is_flag=True,
```

Comment:
You can use a flag that accepts both positive and negative here:

```suggestion
    o = click.option('--mouse/--no-mouse', is_flag=True, default=True
```

This changes the name of this value to `mouse` and avoids a double negation when handling it below. E.g.: `--mouse` is `True` and `--no-mouse` is `False`.

Reply:
nice! learning something new everyday :)